docs/README.md

```diff
@@ -113,7 +112,11 @@ For documentation on the Glowkit API (an updated Bukkit for Glowstone, based on
 
 ## Contributing
 
-First of all, thank you for your interest in advancing Glowstone! We always love to see new developers work on the project! You can find all of our resources on how to get started on our [wiki](https://github.com/GlowstoneMC/Glowstone/wiki#contributing).
+First of all, thank you for your interest in advancing Glowstone! We always love to see new developers work on the project!
+
+[Here](https://github.com/orgs/GlowstoneMC/projects/7) you can find a list of all the remaining features to complete and their associated bounties!
```

Comment:
I think we should contextualize this link in a different way, since it is specifically for 1.13 work.

Reply:
How would like this to be titled / divided ?

Reply:
I guess there are a couple ways to find ideas of what to work on:

- explore the main GitHub issues: https://github.com/GlowstoneMC/Glowstone/issues
- explore the 1.13 board for helping us get up to date to 1.17+
- start the server and play around to find missing features/bugs

Maybe this could be written as 2-3 sentences.